msal/src/test/java/com/microsoft/identity/client/e2e/tests/mocked/SingleAccountOverloadsMockedTest.java

```diff
@@ -350,6 +354,58 @@ public void onError(@NonNull MsalException exception) {
         RoboTestUtils.flushScheduler();
     }
 
+    @Test
+    public void testCanAcquireTokenSilentlyAfterDeviceCode() {
+        // Test sometimes fails because the url is sometimes matched to the token url in MOCK_TOKEN_URL_REGEX
+        // Test case will clear all previous matchers in HttpsClient and reload them as test goes on.
+        mockHttpClient.uninstall();
```

Comment:
Probably put this into an @Before / @After.

Reply:
This is only necessary for this one test case, though

Reply:
+1 to what Adam said, I think it would be nice to do this in before / after regardless of how many tests are using it

Reply:
I may not be understanding this correctly. Let's say setup() is the first @before method ran before each test case. If I add the mockHttpClient.uninstall() call to setup() or create a new @before method for the uninstall, it would get rid of the http intercepts created in setup(), would it not? And what would I be putting in an @after method?

Reply:
I think we can create an `after` method for `uninstall` - but I wouldn't block the PR on this